README.md

```diff
@@ -121,10 +120,10 @@ sections in the book for more details.
 
 You can also build Zebra with additional [Cargo features](https://doc.rust-lang.org/cargo/reference/features.html#command-line-feature-options):
 
-- `sentry` for [Sentry monitoring](https://zebra.zfnd.org/user/requirements.html#sentry-production-monitoring)
-- `journald` for [`journald` logging](https://zebra.zfnd.org/user/tracing.html#journald-logging)
-- `prometheus` for [Prometheus metrics](https://doc.zebra.zfnd.org/zebrad/#metrics)
 - `getblocktemplate-rpcs` for [mining support](https://zebra.zfnd.org/user/mining.html)
+- `prometheus` for [Prometheus metrics](https://zebra.zfnd.org/user/metrics.html)
+- `progress-bar` [experimental progress bars](https://zfnd.org/experimental-zebra-progress-bars/)
+- `sentry` for [Sentry monitoring](https://zebra.zfnd.org/user/tracing.html#sentry-production-monitoring)
```

Comment:
Optional: 

```suggestion
- `sentry` for [Sentry monitoring](https://zebra.zfnd.org/user/tracing.html#sentry-production-monitoring)
- `elasticsearch` for [experimental Elasticsearch support](https://zebra.zfnd.org/user/elasticsearch.html)
```

Reply:
Nice, I forgot about this one.

Reply:
I just realized I forgot to add that feature to the list above because it's not in https://doc.zebra.zfnd.org/zebrad/index.html#zebra-feature-flags. Do you want to add it there? If not, I can add it, but I'll just have to copy-paste the description from the docs.

Reply:
Thanks!, i pushed a quick PR for that [here](https://github.com/ZcashFoundation/zebra/pull/7568).